### DISCUSSION TITLE
New LUA Error

### DISCUSSION THREAD

**Discussion by rathos91** [Gennoken/GennUI#46] [Q&A]
I guess linked to the new ELVui update:

Message: ...nterface/AddOns/ElvUI_GennUI/core/global_private.lua:30: attempt to index field 'queueStatus' (a nil value)
Time: Sun Jun 11 10:07:53 2023
Count: 1
Stack: ...nterface/AddOns/ElvUI_GennUI/core/global_private.lua:30: attempt to index field 'queueStatus' (a nil value)
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_GennUI/core/global_private.lua"]:30: in function `SetupGlobalPrivate'
[string "@Interface/AddOns/ElvUI_GennUI/core.lua"]:57: in function `SetupLayout'
[string "@Interface/AddOns/ElvUI_GennUI/init.lua"]:74: in function <Interface/AddOns/ElvUI_GennUI/init.lua:74>

Locals: (*temporary) = <function> defined =[C]:-1

**Comment by Gennoken** [comment]
fixed
